pkg/image/api/dockertypes.go

```diff
@@ -55,12 +57,19 @@ type DockerConfig struct {
 
 // DockerImageManifest represents the Docker v2 image format.
 type DockerImageManifest struct {
-	SchemaVersion int             `json:"schemaVersion"`
-	Name          string          `json:"name"`
-	Tag           string          `json:"tag"`
-	Architecture  string          `json:"architecture"`
-	FSLayers      []DockerFSLayer `json:"fsLayers"`
-	History       []DockerHistory `json:"history"`
+	SchemaVersion int    `json:"schemaVersion"`
+	MediaType     string `json:"mediaType,omitempty"`
+
+	// schema1
+	Name         string          `json:"name"`
+	Tag          string          `json:"tag"`
+	Architecture string          `json:"architecture"`
+	FSLayers     []DockerFSLayer `json:"fsLayers"`
+	History      []DockerHistory `json:"history"`
+
+	// schema2
+	Layers []distribution.Descriptor `json:"layers"`
```

Comment:
So these are new.  Anything I should worry about.  As I recall, this matched some sort of docker type.